checks/check_status.py

```diff
@@ -127,7 +127,8 @@ def validate_api_key(config):
         if proxy:
             request_proxy = {'https': "http://{user}:{password}@{host}:{port}".format(**proxy)}
         r = requests.get("%s/api/v1/validate" % config['dd_url'].rstrip('/'),
-            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3)
+            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3,
+                         verify=(not config['skip_ssl_validation']))
```

Comment:
Also, I just tested this, and when `requests` is passed `verify=False`, it already throws a warning:

    /opt/datadog-agent/embedded/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:838: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/security.html InsecureRequestWarning)

Do you still want the extra warning? It seems superfluous since requests is now throwing this warning.